tests/batch/test_get_batch_status.py

```diff
@@ -0,0 +1,184 @@
+"""
+Provide tests for command line interface's get batch status command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_batch_status():
+    """
+    Case: get a batch status by its identifier.
+    Expect: batch status is returned.
+    """
+    batch_id = '6f200995e766da7218ec2a3d0aeabbe1151128063cdf4e954cd08390a879b28e' \
+               '085a06f8708d2e6bb34f6501e8ddc981f0353627c1d4f90c80a656a8090c8751'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-status',
+        '--id',
+        batch_id,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert isinstance(json.loads(result.output), dict)
+
```

Comment:
Add check for the return value (if the value has a string format and exists in (`COMMITTED`, ...)).

Reply:
Done.